CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 - feat: Async callstacks are being tracked by wrapping the `dispatch_async` and related APIs. #998
 - feat: Add transaction to the scope #992
+- fix: Pass SentryTracer to span child #1040
+- feat: Add span to SentrySDK #1042
```

Comment:
2 entries? I believe `1040` should be removed

Reply:
I forgot to add the 1040 in the last PR. So, I updated it now. It is a problem?

Reply:
all good, you could also edit it directly without a PR for such cases, its also fine.